README.md

```diff
@@ -1,13 +1,29 @@
-SymfonyMakerBundle
-==================
+The Symfony MakerBundle
+=======================
 
-Symfony Maker helps you creating empty commands, controllers, form classes,
-tests and more so you can forget about the required boilerplate code. This
-bundle is an alternative to [SensioGeneratorBundle][1] for modern Symfony
-applications and requires using Symfony 3.4 or newer and [Symfony Flex][2].
+The MakerBundle is the fastest way to generate the most common code you'll
+need in a Symfony app: commands, controllers, form classes, event susbcribers
+and more! This bundle replaces [SensioGeneratorBundle][1] for modern Symfony
+applications and requires Symfony 3.4 or newer and [Symfony Flex][2].
 
-[Read the docs][3]
+[![Build Status](https://travis-ci.org/knpuniversity/oauth2-client-bundle.svg)](http://travis-ci.org/symfony/maker-bundle)
```

Comment:
Looks like something is wrong here :)

Reply:
But anyway, I would remove this build status that does not give any useful information for people reading the README.